plugins/doc_fragments/mysql.py

```diff
@@ -79,8 +79,10 @@ class ModuleDocFragment(object):
      yum install python2-PyMySQL (RHEL/CentOS/Fedora; see M(ansible.builtin.yum)). You can also use dnf install python2-PyMySQL
      for newer versions of Fedora; see M(ansible.builtin.dnf).
    - Be sure you have PyMySQL or MySQLdb library installed on the target machine
-     for the Python interpreter Ansible uses, for example, if it is Python 3,
-     you must install the library for Python 3. You can also change the interpreter.
+     for the Python interpreter Ansible discovers.  For example if ansible discovers and uses Python 3, you need to install
+     the Python 3 version of PyMySQL.  If ansible discovers and uses Python 2, you need to install the Python 2 version
+     of either PyMySQL or MySQL-python.
+   - If you have trouble, it may help to force Ansible to use Python 3 by specifying C(ansible_python_interpreter).
```

Comment:
```suggestion
   - If you have trouble, it may help to force Ansible to use a Python interpreter you need by specifying the C(ansible_python_interpreter) inventory variable.
```